src/modules/blockchainApplication/manage/components/AddApplicationList/AddApplicationList.js

```diff
@@ -5,15 +5,17 @@ import Box from '@theme/box';
 import BoxHeader from '@theme/box/header';
 import BoxContent from '@theme/box/content';
 import { useBlockchainApplicationExplore } from '@blockchainApplication/explore/hooks/queries/useBlockchainApplicationExplore';
-import { QueryTable } from 'src/theme/QueryTable';
+import { QueryTable } from '@theme/QueryTable';
+import useMergeApplicationExploreAndMetaData from '../../hooks/useMergeApplicationExploreAndMetaData';
 import { useSearchApplications } from '../../hooks/useSearchApplications';
 import AddApplicationSearch from '../AddApplicationSearch/AddApplicationSearch';
 import AddApplicationRow from '../AddApplicationRow/AddApplicationRow';
 import styles from './AddApplicationList.css';
 
 const AddApplicationList = () => {
   const { t } = useTranslation();
-  const { data, ...searchResponse } = useSearchApplications();
+  const { ...searchResponse } = useSearchApplications();
+  const { isURL, URLStatus, debouncedSearchValue } = searchResponse;
```

Comment:
I guess this is unnecessary i guess this would be sufficient except if `searchResponse` is used somewhere else which I don't see in code

```suggestion
  const { isURL, URLStatus, debouncedSearchValue} = useSearchApplications();
```

Reply:
@eniolam1000752 `searchResponse` is passed into the `AddApplicationSearch` component like this `<AddApplicationSearch {...searchResponse} />` because some other properties of it are used there. So the implementation might not change to what you suggested